src/reporting/clients/IMappingsClient.ts

```diff
@@ -0,0 +1,537 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { AccessToken } from "@itwin/core-bentley";
+import type { CalculatedProperty, CalculatedPropertyCreate, CalculatedPropertyUpdate } from "../interfaces/CalculatedProperties";
+import type { CustomCalculation, CustomCalculationCreate, CustomCalculationUpdate } from "../interfaces/CustomCalculations";
+import type { GroupProperty, GroupPropertyCreate, GroupPropertyUpdate } from "../interfaces/GroupProperties";
+import type { Group, GroupCreate, GroupUpdate } from "../interfaces/Groups";
+import type { Mapping, MappingCopy, MappingCreate, MappingUpdate } from "../interfaces/Mappings";
+import type { EntityListIterator } from "../iterators/EntityListIterator";
+
+export interface IMappingsClient {
+  /**
+   * Gets all Mappings for an iModel. This method returns the full list of mappings.
+   * @param {string} iModelId The iModel Id.
+   * @param {string} accessToken OAuth access token with scope `insights:read`
+   * @param {number} top The number of entities to load.
+   * @memberof ReportingClient
+   * @link https://developer.bentley.com/apis/insights/operations/get-mappings/
+   */
+  getMappings(
+    accessToken: AccessToken,
+    iModelId: string,
+    top?: number
+  ): Promise<Mapping[]>;
+
+  /**
+   * Gets an async paged iterator of Mappings for an iModel.
+   * This method returns an iterator which loads pages of mappings as it is being iterated over.
+   * @param {string} iModelId The iModel Id.
+   * @param {string} accessToken OAuth access token with scope `insights:read`
+   * @param {number} top The number of entities to load.
+   * @memberof ReportingClient
+   * @link https://developer.bentley.com/apis/insights/operations/get-mappings/
+   */
+  getMappingsIterator(
+    accessToken: AccessToken,
+    iModelId: string,
+    top?: number
+  ): EntityListIterator<Mapping>;
+
+  /**
+   * Gets a Mapping for an iModel.
+   * @param {string} iModelId The iModel Id.
+   * @param {string} mappingId The Mapping Id.
+   * @param {string} accessToken OAuth access token with scope `insights:read`
+   * @memberof ReportingClient
+   * @link https://developer.bentley.com/apis/insights/operations/get-mapping/
+   */
+  getMapping(accessToken: AccessToken,
```

Comment:
Inconsistent formatting - move `accessToken` to new line. `getMappingsIterator` is formatted correctly. Use that as an example.

Reply:
fixed all such problems